### ISSUE TITLE
When using versionbot, wait until versionbot has been merged to finalize the release

### ISSUE THREAD

**Issue by pipex** [balena-io/deploy-to-balena-action#184]
When using versionbot, this is how the action interacts with that service

1. On PR: versionbot creates a branch with the version. The action uses that branch to create the release
2. On merge: bulldozer merges the original branch (without the version bump). The action finalizes the release
3. Versionbot runs one more time, and creates a new commit with the version bump.

If step 3 fails, the release version will get out of sync with the code version (e.g. release may be 2.0.1+rev1, while the repository is still at 2.0.0).

Moving the release finalization to step 3, when `versionbot: true` should address the issue.

**Comment by 20k-ultra** [comment]
https://github.com/balena-io/deploy-to-balena-action/issues/181 would solve this. We could make the trigger a PR like we currently do but make the trigger for finalize a commit by versionbot... I'll have to think about this more but it's pushing it for scope of this action to make up for issues in versionbot I think. (I want to fix it, don't think i'm trying to get off the hook.

**Comment by 20k-ultra** [comment]
I think the easiest and best solution is to have an action that checks if versionbot ran. This new action looks at the main branch and since we know Versionbot commit is always after bulldoze we can just find the latest merge commit in the timeline and look for a versionbot commit after it.

**Comment by pipex** [comment]
Right, I agree #181 would address this. In that scenario, since the versionbot action is configured, the result of the action will be false in the bulldozer merge, but it will be true on the versionbot merge, thus, ensuring the finalize is done at that point.
